components/core/FormContainer.js

```diff
@@ -0,0 +1,35 @@
+const AbstractComponent = require('../AbstractComponent');
+
+/**
+ * The Form Buttom Core Component. In addition to the Component methods, this component provides several 
+ * convience methods to set known properties. 
+ * 
+ * Based on the documentation found at:
+ * https://github.com/adobe/aem-core-wcm-components/tree/master/content/src/content/jcr_root/apps/core/wcm/components/form/container/v2/container
+ */
+
+ class FormContainer extends AbstractComponent {
+
+ 	constructor() {
+ 		super();
+ 		this.props["sling:resourceType"] = "core/wmc/components/form/container/v2/container"
```

Comment:
Correct resourceType path here. Same in several other locations, please check all resourceType for correct pathing